tests/TestHelpers/UtilsTrait.php

```diff
@@ -64,4 +66,26 @@ protected static function getKeyedArray(array $providedData, int $useArgumentNum
         }
         return $newData;
     }
+
+    /**
+     * Change a nested array element.
+     *
+     * @param array $keys
+     *   Ordered keys to the value to set.
+     * @param array $data
+     *   The array to modify.
+     * @param mixed $newValue
+     *   The new value to set.
+     *
+     * @return void
+     */
+    protected function nestedChange(array $keys, array &$data, $newValue) : void
```

Comment:
Not in scope, but nestedChange() is such a weird name for this method. Maybe setNested() or just set() would be preferable?

Reply:
Yeah this is just moving an existing method. but I agree it could be named better